lib/platform.cpp

```diff
@@ -450,10 +450,10 @@ Return Value:
 
 #if defined(_WIN32)
     HasDotProductInstructions = (IsProcessorFeaturePresent(PF_ARM_V82_DP_INSTRUCTIONS_AVAILABLE) != 0);
+#elif defined(__linux__)
```

Comment:
what about `apple`?

Reply:
I check the latest code from onnxruntime, it doesn't consider `apple` either